Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -307,7 +306,7 @@ bool GStreamerMediaEndpoint::initializePipeline()
         GstWebRTCPeerConnectionState state;
         g_object_get(webrtcBin, "connection-state", &state, nullptr);
         GUniquePtr<char> desc(g_enum_to_string(GST_TYPE_WEBRTC_PEER_CONNECTION_STATE, state));
-        auto dotFilename = makeString(unsafeSpan(GST_ELEMENT_NAME(endPoint->pipeline())), '-', unsafeSpan(desc.get()));
+        auto dotFilename = makeString(String::fromUTF8(GST_ELEMENT_NAME(endPoint->pipeline())), '-', String::fromUTF8(desc.get()));
```

Comment:
Connot these two be `CStringView::unsafeFromUTF8()` instead of `String::fromUTF8()` to avoid copies?